src/Integration.Vsix/SonarLintTagger/TaggerProvider.cs

```diff
@@ -160,15 +161,21 @@ private void OnAnalysisRequested(object sender, AnalysisRequestEventArgs args)
 
             if (detectedLanguages.Any() && analyzerController.IsAnalysisSupported(detectedLanguages))
             {
-                var issueTracker = buffer.Properties.GetOrCreateSingletonProperty(typeof(IIssueTracker),
-                    () => new TextBufferIssueTracker(dte, this, textDocument, detectedLanguages, issuesFilter, sonarErrorDataSource, converter, logger));
+                // We only want one TBIT per buffer and we don't want it be disposed until
```

Comment:
As discussed, I've reverted this change to the TaggerProvider/TBIT as it TaggerProvider has additional behaviours that will need more fixing up.